src/routes/__layout-root.svelte

```diff
@@ -63,23 +63,25 @@
   import Banners from '$lib/components/banner/banners.svelte';
   import { ErrorBoundary } from '$lib/components/error-boundary';
   import PageTitle from '$lib/holocene/page-title.svelte';
+  import ScrollToTop from '$lib/holocene/scroll-to-top.svelte';
```

Comment:
Wondering if we should add this and make updates to `__layout-import.svelte` as well 🤔